tests/github-action-helper.sh

```diff
@@ -267,6 +267,27 @@ install_minikube_with_none_driver() {
   sudo -E minikube start --kubernetes-version="$1" --driver=none --memory 6g --cpus=2 --addons ingress --cni=calico
 }
 
+install_external_snapshotter() {
+  curl -L https://github.com/kubernetes-csi/external-snapshotter/archive/refs/tags/v6.2.0.zip -o external-snapshotter.zip
```

Comment:
couple of things
* declare version as a variable that needs update in only one place 
* use the latest 7.0.2 release
* download the folder to temp folder instead of the root from where the tool is running.